addon/serializers/linked-node.js

```diff
@@ -0,0 +1,16 @@
+import OsfSerializer from './osf-serializer';
+
+// TODO Dawn - a linked-node serializer is actually using the node model - Ember doesn't like this.
```

Comment:
Is this TODO still relevant?  Would we need a followup ticket to deal with any issues here?

I see we added a custom serializer.